pkg/resource/function/hooks.go

```diff
@@ -614,63 +621,114 @@ func (rm *resourceManager) setResourceAdditionalFields(
 	}
 	ko.Spec.ReservedConcurrentExecutions = getFunctionConcurrencyOutput.ReservedConcurrentExecutions
 
-	var getFunctionEventInvokeConfigOutput *svcsdk.GetFunctionEventInvokeConfigOutput
-	getFunctionEventInvokeConfigOutput, err = rm.sdkapi.GetFunctionEventInvokeConfigWithContext(
+	return nil
+}
+
+// getFunctionCodeSigningConfig will describe the code signing
```

Comment:
You also forgot to update the function name in this comment

Reply:
Done